src/main/scala/org/camunda/feel/impl/builtin/TemporalBuiltinFunctions.scala

```diff
@@ -29,4 +37,37 @@ class TemporalBuiltinFunctions(clock: FeelEngineClock) {
     }
   )
 
+  private def dateTimeFunction(function: Date => Val): ValFunction =
+    builtinFunction(
+      params = List("date"),
+      invoke = {
+        case List(ValDate(date))          => function(date)
+        case List(ValDateTime(date))      => function(date.toLocalDate)
+        case List(ValLocalDateTime(date)) => function(date.toLocalDate)
+      }
+    )
+
+  private def getDayOfYear(date: Date): ValNumber = {
+    val dayOfYear = date.getDayOfYear
+    ValNumber(dayOfYear)
+  }
+
+  private def getDayOfWeek(date: Date): ValString = {
+    val dayOfWeek = date.getDayOfWeek
+    val displayName = dayOfWeek.getDisplayName(TextStyle.FULL, Locale.ENGLISH)
```

Comment:
Did you consider overloading the function with an additional Locale parameter (or a String that can be parsed to a Locale)?

Reply:
Good idea :+1: 
But let's wait until someone requests it :smiling_imp: